registrations/tests/test_signup_group_post.py

```diff
@@ -761,20 +761,53 @@ def test_signup_group_date_of_birth_is_mandatory_if_audience_min_or_max_age_spec
 
 
 @pytest.mark.parametrize(
-    "date_of_birth,expected_status,expected_error",
+    "date_of_birth,expected_status,expected_error,has_event_start_time",
     [
-        ("2011-04-07", status.HTTP_400_BAD_REQUEST, "The participant is too young."),
-        ("1879-03-14", status.HTTP_400_BAD_REQUEST, "The participant is too old."),
-        ("2000-02-29", status.HTTP_201_CREATED, None),
+        (
+            "2004-03-13",
+            status.HTTP_400_BAD_REQUEST,
+            "The participant is too young.",
+            False,
+        ),
+        (
+            "2004-03-13",
+            status.HTTP_201_CREATED,
+            None,
+            True,
+        ),
+        (
+            "1982-03-13",
+            status.HTTP_400_BAD_REQUEST,
+            "The participant is too old.",
+            False,
+        ),
+        (
+            "1983-03-13",
+            status.HTTP_400_BAD_REQUEST,
+            "The participant is too old.",
+            True,
+        ),
+        ("2000-02-29", status.HTTP_201_CREATED, None, False),
     ],
 )
 @freeze_time("2023-03-14 03:30:00+02:00")
 @pytest.mark.django_db
 def test_signup_group_age_has_to_match_the_audience_min_max_age(
```

Comment:
There is a similar test for the `signup` endpoint (`test_signup_age_has_to_match_the_audience_min_max_age`) - that one should be updated as well

Reply:
Updated also tests for the `signup` endpoint